Makefile

```diff
@@ -34,8 +34,8 @@ endif
 #
 
 # keep standard at C11 and C++11
-CFLAGS   = -I.              -O3 -DNDEBUG -std=c11   -fPIC
-CXXFLAGS = -I. -I./examples -O3 -DNDEBUG -std=c++11 -fPIC
+CFLAGS   = -I.              -O3 -DNODEBUG -std=c11   -fPIC
+CXXFLAGS = -I. -I./examples -O3 -DNODEBUG -std=c++11 -fPIC
```

Comment:
I don't think this is right, `NDEBUG` is used by standard C to disable asserts and if you change it to `NODEBUG` it won't achieve that any more.

Reply:
That's a typo. Will fix.